# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bonobo10/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur bonobo-1.0.22-orig/tests/test-any.c bonobo-1.0.22/tests/test-any.c
--- bonobo-1.0.22-orig/tests/test-any.c	2002-01-28 10:54:07.000000000 +0000
+++ bonobo-1.0.22/tests/test-any.c	2006-01-18 17:17:09.087366250 +0000
@@ -15,18 +15,18 @@
 	TC_octet
 };
 
-static const struct CORBA_TypeCode_struct
-TC_CORBA_sequence_CORBA_octet_struct = {
-      {{(ORBit_RootObject_Interface *) & ORBit_TypeCode_epv, TRUE, -1},
-       ORBIT_PSEUDO_TYPECODE},
-
-      CORBA_tk_sequence, NULL, NULL,
-      0, 1,
-      NULL,
-      (CORBA_TypeCode *) octet_subtypes_array,
-      NULL,
-      CORBA_OBJECT_NIL, 0, -1, 0, 0
-};
+//static const struct CORBA_TypeCode_struct
+//TC_CORBA_sequence_CORBA_octet_struct = {
+//      {{(ORBit_RootObject_Interface *) & ORBit_TypeCode_epv, TRUE, -1},
+//       ORBIT_PSEUDO_TYPECODE},
+//
+//      CORBA_tk_sequence, NULL, NULL,
+//      0, 1,
+//      NULL,
+//      (CORBA_TypeCode *) octet_subtypes_array,
+//      NULL,
+//      CORBA_OBJECT_NIL, 0, -1, 0, 0
+//};
 
 static const char *
 enum_subnames_array [] = {
